tests/utils/utils.go

```diff
@@ -323,25 +451,23 @@ func calculateTxParams(ctx context.Context, wsClient ethclient.Client, fundedAdd
 	return gasFeeCap, gasTipCap, nonce
```

Comment:
on line 449: where does the `2500000000` come from?

Reply:
Replaced with `gasUtils.MaxPriorityFeePerGas`